cmd/registry/cmd/annotate.go

```diff
@@ -0,0 +1,258 @@
+// Copyright 2021 Google LLC. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"strings"
+
+	"github.com/apigee/registry/cmd/registry/core"
+	"github.com/apigee/registry/connection"
+	"github.com/apigee/registry/gapic"
+	"github.com/apigee/registry/rpc"
+	"github.com/apigee/registry/server/names"
+	"github.com/spf13/cobra"
+	"google.golang.org/genproto/protobuf/field_mask"
+)
+
+const annotateFieldName = "annotations"
+const annotateCommandName = "annotate"
+
+func init() {
+	rootCmd.AddCommand(annotateCmd())
+}
```

Comment:
@seaneganx Just reiterating that I think it would be good for future refactoring to eliminate use of init to build the command hierarchy. It's a bit more verbose, but I think it would be better for the parent command to explicitly add its children, e.g. the `rootCmd()` constructor would include line 36 to add the annotate subcommand.